### ISSUE TITLE
500 Error on local Deploy

### ISSUE THREAD

**Issue by StirStudios** [drunomics/lupus-decoupled-nuxt3-demo#6]
**Nuxt**
"nuxt": "^3.4.2"
"nuxtjs-drupal-ce": "^2.0.0-alpha9"

**Drupal**
"drupal/core-recommended": "^10.8"
"drupal/lupus_decoupled": "1.x-dev@dev"

**Drupal Config**
_Frontend Base URL_
http://localhost:3000

**Nuxt Config**
```
export default defineNuxtConfig({
  modules: ["nuxtjs-drupal-ce"],
  drupalCe: {
    baseURL: "http://localhost:49179/ce-api",
  },
  nitro: {
    compressPublicAssets: true,
  },
});
```

However all pages give a 500 error:

```
fetch failed (http://localhost:49179/ce-api/node/3)
at Module.createError (./node_modules/nuxt/dist/app/composables/error.js:37:38)
at fetchPage (./node_modules/nuxtjs-drupal-ce/dist/runtime/composables/useDrupalCe.mjs:38:35)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
at async setup (./pages/[...slug].js:21:197)
```

Nothing in the Drupal logs.

Launch it on StackBlitz, gives an error as well...

```
request to http://lupus-decoupled.ddev.site/ce-api failed, reason: socket hang up (http://lupus-decoupled.ddev.site/ce-api)
at createError (./node_modules/h3/dist/index.mjs:229:15)
at createError (./node_modules/nuxt/dist/app/composables/error.js:37:38)
at fetchPage (./node_modules/nuxtjs-drupal-ce/dist/runtime/composables/useDrupalCe.mjs:38:35)
```

Any ideas as to what might be going on?

**Comment by StirStudios** [comment]
Problem solved if you update localhost -> 127.0.0.1. This is actually something quite common with Vite and Nuxt 3 as far as what I have seen.

**Comment by fago** [comment]
Interesting, thank you for sharing!
Let's close this then.
